packages/daemon/daemon-docs.md

```diff
@@ -1,26 +1,29 @@
-# `@endo/cli`
+# `@endo/daemon`
 
-### `endo start`
+The Endo Daemon allows us to bootstrap systems in which POLA confinement can be accomplished at the deepest level of granularity. In the case of an Endo application, the deepest level of granularity refers to the lexical environment of each object that exists within the system. This is otherwise known as the "object level".
 
-- executes [stop](#start) function definined within the `@endo/daemon`.
+Endo's approach differs from systems that seek to enforce security at the "application level". Confinement at application-level requires an attacker to bypass just one-level of a program in order to successfully gain access to all of the powers that it holds. Confinement at the object-level forces the attacker to break the security of each object directly to successfully gain access to all of the powers that a program requires to operates.
 
-### `endo stop`
+## Endo as a Powerbox
 
-- executes [stop](#stop) function definined within the `@endo/daemon`.
-
-### `endo clean`
+Endo makes use of the Powerbox pattern[^1]. By employing this pattern, Endo acts as a mediator between all of the objects in a system, past, present, and future, and the powers that an application may ever need in order to operate.
 
-- executes the [`clean`](#clean) function definined within the `@endo/daemon`.
+> **The powerbox is a composition of objects that grants, revokes, negotiates, and in general manages, the authorities granted to another object.** - _How Emily Tamed the Caml_
 
-### `endo restart`
+## Methods
 
-- shorthand method for executing `endo stop`
+Upon launch of the "main" module, the Endo daemon is comprised of the following powers:
 
-# `@endo/daemon`
+- `crypto`
+- `net`
+- `fs`
+- `path`
+- `popen`
+- `url`
```

Comment:
This is a very Node.js specific list of powers needed in order to construct the powers actually needed by the daemon. Those are defined by `DaemonicPowers` in `types.d.ts`.